build/parseChangelog.c

```diff
@@ -185,6 +184,8 @@ static int dateToTimet(const char * datestr, time_t * secs, int * date_words)
     rc = 0;
 
 exit:
+    if (tz_name)
```

Comment:
Don't bother testing for NULL, free() does it anyway so it ends up being just useless clutter.